### DISCUSSION TITLE
About CSS variables

### DISCUSSION THREAD

**Discussion by AndreyDmitriev39r** [jdwilkin4/fcc-forum-clone#26] [Ideas]
I inspected some codebases, and I saw some folks are using CSS variables not only for colors, but for many things: width, border-radius, box-shadow, transition... Pretty much all kind of things.

Do we stick to CSS variables only for colors, or maybe will use them for more CSS properties? I do not have opinion about that, and will be ok with every option.

**Comment by jdwilkin4** [comment]
For this project, we don't really have that much css to worry about. Besides the navbar, we just have a table and some headers and table data. 

If this were a larger project, or production site, then yes you can definitely use CSS variables for other things besides colors. In those situations you would have multiple stylesheets and would want to develop some consistency throughout the application. In most situations, you would already be jumping into a larger codebase that had all of that structure laid out for you.  Or you would be using a CSS library and framework like tailwind or chakra UI that used its own classes for styling.
